packages/cli/src/commands/push.ts

```diff
@@ -94,20 +91,9 @@ async function runActionIfRequested(externalOptions: any): Promise<void> {
 }
 
 async function runActionIfNeeded(contractName: string, network: string, options: any): Promise<void> {
-  const { force, interactive, network: promptedNetwork } = options;
-  const projectFile = new ProjectFile();
-  const networkFile = new NetworkFile(projectFile, network);
-  const { contract: contractAlias, package: packageName } = fromContractFullName(contractName);
-
-  if (interactive) {
-    if (
-      force ||
-      (!packageName && !networkFile.hasContract(contractAlias)) ||
-      (packageName && !networkFile.hasDependency(packageName))
-    ) {
-      await action({ ...options, dontExitProcess: true, skipTelemetry: true });
-    }
-  }
+  const { interactive } = options;
+  if (!interactive) return;
```

Comment:
I personally prefer 1) avoiding early returns, and 2) always using braces. The latter should be a linter rule if we even agree to it, so I won't insist on it.

Reply:
1) I like returns either at the beginning or the end, though not in the middle. Returns at the very first lines of the function feel natural as they are kind of "validations" of whether we want to run or not. Agree on avoiding them in the middle.

2) I particularly line single-line ifs when it's a return. Seems like we're on opposite sides on this one!

Reply:
Actually, just changed this so the return is on the very first line.